deeplake/core/chunk_engine.py

```diff
@@ -1463,13 +1467,13 @@ def _check_rechunk(self, chunk: BaseChunk, chunk_row: int):
             chunk.num_data_bytes < RANDOM_MINIMAL_CHUNK_SIZE
             and self.max_chunk_size > RANDOM_MINIMAL_CHUNK_SIZE
         ):
-            self._try_merge_with_neighbor_and_split(chunk=chunk, row=chunk_row)
+            return self._try_merge_with_neighbor_and_split(chunk=chunk, row=chunk_row)
 
         elif (
             chunk.num_data_bytes > RANDOM_MAX_ALLOWED_CHUNK_SIZE
             or chunk.num_data_bytes > self.max_chunk_size + RANDOM_MINIMAL_CHUNK_SIZE
         ):
-            self.__rechunk(chunk, chunk_row)
+            return self.__rechunk(chunk, chunk_row)
```

Comment:
why? it doesn't return anything but None

Reply:
same here